src/main.js

```diff
@@ -59,45 +49,49 @@ function randomCover(){
     var randomDescOne = descriptors[getRandomIndex(descriptors)]
     var randomCover = covers[getRandomIndex(covers)]
     var randomTitle = titles[getRandomIndex(titles)]
-   currentCover([randomTitle, randomCover, randomDescOne, randomDescTwo])
-   //savedCovers.push(randCover);
+   makeCurrentCover([randomTitle, randomCover, randomDescOne, randomDescTwo])
 }
 randomCover()
 
 function changeToSavedCovers() {
-  homeButton.classList.remove('hidden');
-  saveCoverButton.classList.add('hidden');
-  randomCoverButton.classList.add('hidden');
-  mainCover.classList.add('hidden');
+  changePage(2)
   savedCoversSection.classList.remove('hidden')
-  formView.classList.add('hidden');
 };
 
 function changeToMakeNewCover() {
-  homeButton.classList.remove('hidden');
-  saveCoverButton.classList.add('hidden');
-  randomCoverButton.classList.add('hidden');
-  mainCover.classList.add('hidden');
-  savedCoversSection.classList.add('hidden');
+  changePage(3)
   formView.classList.remove('hidden')
 };
 
 function goHome() {
+  changePage(1)
   saveCoverButton.classList.remove('hidden')
   homeButton.classList.add('hidden');
   randomCoverButton.classList.remove('hidden')
   mainCover.classList.remove('hidden')
-  savedCoversSection.classList.add('hidden');
-  formView.classList.add('hidden');
 };
 
-function makeBookCover() {
+function changePage(num){
```

Comment:
Nice This is a smart use of a helper function.

Reply:
I am curious as to why toggle wasn't utilized, though. But I like the work around.

Reply:
We originally were using an entirely different method to hide/unhide everything and more recently converted to classList.Add/Remove. The syntax of that earlier method still lives on in our code (and I can't quite think through how to toggle properly between three different buttons).

Reply:
Yes I was having the same issue when working with .toggle()